docs/rst/manual/experimental/webviz-ert.rst

```diff
@@ -0,0 +1,96 @@
+Webviz-ert
+==========
+
+
+.. DANGER::
+   Webviz-ert and the corresponding ert-api is experimental, meaning that the interface
+   and/or API are subject to breaking changes. In addition, such breaking changes might
+   not follow rules for semantic versioning, as the feature is considered experimental
+   and not part of the public interface.
+
+
+Introduction
+------------
+
+Webviz-ert is what we consider the future of visualisation in ERT, and the old
+plotting tool based on PyQt will at some point be deprecated and eventually removed.
+Our main goal is to provide our users with a more rich interface with the common, yet
+simple visualisations needed. Advanced users / needs would still be pointed in the
+direction of ResInsight or equivalent.
+
+Feedback regarding stability, bugs and features is very much appreciated.
```

Comment:
Perhaps write how to contact us / slack #ert-users / yammer / etc

Reply:
i think it is stated in the ert docs in general - so i guess it should not be needed for this page explicitly?